README.md

```diff
@@ -1,9 +1,5 @@
 # Kueue
-
-Kueue is a set of APIs and controller for [job](docs/concepts/workload.md)
-[queueing](docs/concepts#queueing). It is a job-level manager that decides when
-a job should be [admitted](docs/concepts#admission) to start (as in pods can be
-created) and when it should stop (as in active pods should be deleted).
+[Overview](https://kueue.sigs.k8s.io/docs/overview/)
 
 ## Why use Kueue
```

Comment:
Maybe we can just point to the page, similar to the `#Kueue` section.
@alculquicondor WDYT?

Reply:
It makes sense.